stack.yaml

```diff
@@ -1 +1 @@
-resolver: lts-7.14
```

Comment:
The project was no-longer buildable under this, so I updated it to a fairly recent one.